src/api/procedures/__tests__/issueTokens.ts

```
@@ -134,7 +134,7 @@ describe('issueTokens procedure', () => {
     );
   });
 
-  test('shoud throw an error if at least one identity id has not a canTransfer success status', () => {
+  test('shoud throw an error if at least one identity id has not a canMint success status', () => {
```

Should throw an error if canMint returns a status different from Success